IBAnimatable/TransitionAnimationType.swift

```diff
@@ -190,6 +190,32 @@ extension TransitionAnimationType {
         return .cross
       }
     }
+
+    var matchingGesture: InteractiveGestureType.GestureDirection {
+      switch self {
+      case .left: return .left
+      case .right: return .right
+      case .top: return .top
+      case .bottom: return .bottom
+      case .forward: return .open
+      case .backward: return .close
+      default:
+        fatalError("No matching GestureDirection for \(self)")
+      }
+    }
+
+    var opposingGesture: InteractiveGestureType.GestureDirection {
+      switch self {
+      case .left: return .right
+      case .right: return .left
+      case .top: return .bottom
+      case .bottom: return .top
+      case .forward: return .close
```

Comment:
This property goes from 
`TransitionAnimationType.Direction` -> `InteractiveGestureType.GestureDirection`
`.forward` -> `.close`
`.backward` -> `.open`

To go the other way I would have to add a property to `InteractiveGestureType.GestureDirection`

Which at the moment would be a superfluous property as we don't use this in the codebase